sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -138,8 +143,25 @@ def as_dict(self):
 class CounterMetric(Metric):
   def __init__(self, counter_dict, submit_timestamp, metric_id):
     super(CounterMetric, self).__init__(submit_timestamp, metric_id)
-    self.value = counter_dict.committed
     self.label = str(counter_dict.key.metric.name)
+    self.value = counter_dict.committed
+
+
+class DistributionMetrics(Metric):
+  def __init__(self, dists, submit_timestamp, metric_id, key):
```

Comment:
also do multiple dists have the same dist label? I also wonder what is the relationship between label in metric_id - is there 1-1 correspondence?

Reply:
Renaming it sounds reasonable. I'm changing it this way.

Your comment made me think that I may be wrong by assuming there is no problem if someone would add measurements to the same distribution several times during one pipeline. In BQ they would be redundantly saved with same `id`, `label` and `timestamp` but with different `value`. It would be good to differ them somehow to know how a saved distribution looks like.